calico-enterprise/reference/installation/tigerastatus.mdx

```diff
@@ -0,0 +1,42 @@
+---
+description: Descriptions of Tigera Status fields
+---
+
+# Tigera status
+
+Installing {{prodname}} on your Kubernetes cluster is managed by the Tigera operator. The Tigera operator is deployed as a ReplicaSet in the `tigera-operator` namespace, and records status in a custom resource named, `tigerastatus`. The operator get its configuration from several Custom Resources (CRs); the central one is the Installation CR.
+
+To check tigerastatus, use the following command.
+
+`kubectl get tigerastatus`
+
+## Log storage controllers
+
+The following controllers comprise the log storage component. Note the following:
+
+- All controllers are required and are not configurable
+- Controllers work together to implement log storage functionality, but are reconciled independently
+- Controllers do not communicate directly with each other; they communicate through the Kubernetes API
+
+To check log collector status, run the following command:
+
+`kubectl get logcollector.operator.tigera.io tigera-secure`
+
+The following table lists controller descriptions.
+
+| Tigera status field         | Description                                                  |
+| --------------------------- | ------------------------------------------------------------ |
+| log-storage                 | Provides initialization validation and defaulting for log storage, creates base namespaces for controllers to deploy to, and provides status and condition updates on other controllers. |
```

Comment:
I think instead of wording the descriptions in terms of what the controller does, we should work them in terms of what the TigeraStatus in the first column reports on.

e.g., 

- "Displays overall status of LogStorage initialization and validation"
- "Displays status of the Elasticsearch cluster and Kibana"

etc.

We might also want to add additional columns for things like when this TigeraStatus is expected.

| TigeraStatus name | Description | Standalone | Managed | Management |
|---------------------------|-----------------|--------------|---------------|--------------|
| log-storage | Displays overall log storage subsystem status | &#x2611; | | &#x2611; |

wdyt?

Reply:
I like it. Added.